src/GraphQL/Resolvers/FieldResolverExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace GraphQL.Resolvers
+{
+    public static class FieldResolverExtensions
+    {
+        public static async Task<object> ResolveAsync(this IFieldResolver resolver, IResolveFieldContext context)
```

Comment:
We might think of a better name for this method.

Reply:
`ResolveAsync` seems fine to me.